src/OpenLoco/src/Map/MapGenerator/MapGenerator.cpp

```diff
@@ -416,6 +411,9 @@ namespace OpenLoco::World::MapGenerator
 
         CompanyManager::setUpdatingCompanyId(CompanyId::neutral);
 
+        auto progressTicksPerIndustry = (maxProgress - minProgress) / numIndustriesAvailable;
```

Comment:
Is there the possibility that `numIndustriesAvailable` can be zero here?

Reply:
Nice catch. Yes, it ranges from 0-2. I should handle this differently.

Reply:
Nice catch. Yes, though rarely, I should exit early in that case, I suppose.